source/index.html.md

```diff
@@ -798,6 +871,58 @@ Only one 3-D Secure version can be used for a given authorization.
   </dd>
 </dl>
 
+##### Scheme reference to series
+
+If the previous-in-series authorization was made via this API, you should use
+`series[previous][id]` to reference the previous-in-series authorization.
+Otherwise, you must obtain explicit approval from Clearhaus to use these raw
+scheme values grouped in <code>series[previous][mastercard]</code> and
+<code>series[previous][visa]</code>.
+
+The Mastercard specific reference to the series contains the following parts.
+
+<dl class="dl-vertical">
+  <dt>
+    series[previous][mastercard][tid]
+    <span class="type">[A-z0-9]{3}[A-z0-9]{6}[0-9]{4}[ ]{2}</span>
+  </dt>
+  <dd>
+    Trace ID being the concatenation of values
+    Data element 63 subfield 1 (Financial Network Code) (position 1-3),
+    Data element 63 subfield 2 (Banknet Reference Number) (position 4-9),
+    Data element 15 (Date, Settlement) (position 10-13), and
+    two spaces;
+    to be used in Data Element 48, Subfield 63.
+
+    <div class="type">Optional. Cannot be present if <code>series[previous][id]</code> or <code>series[previous][visa]</code> is present.</div>
+  </dd>
+
+  <dt>
+    series[previous][mastercard][tokenized]
+    <span class="type">(true|false)</span>
+  </dt>
+  <dd>
+    Indicate if the series was initiated with a tokenized payment method.
```

Comment:
We might need to add some additional explanation.

Reply:
I didn't think about that, but you're right, we definitely do!

I'm in doubt if it should it go here or in a guideline — WDYT?

Reply:
I lean towards putting it in a guideline rather than having it here.

Are we sure that "tokenized payment method" is correct wording? For instance, Google Pay, a payment method, can be both tokenized and non-tokenized :man_shrugging:

Reply:
> Are we sure "tokenized payment method" is correct wording? For instance, Google Pay, a payment method, can be both tokenized and non-tokenized 🤷‍♂️

Luckily the current documented version (on `master`) doesn't support Google Pay 😄

Though, I think we should just word it like this:
```suggestion
    Indicate if the series was initiated with a tokenized payment.
```

Reply:
I think that works well :+1: